Hivemind/Assets/Scripts/UI/GameMenuScript.cs

```diff
@@ -42,6 +48,20 @@ public void PauseGame()
         TimeController.Instance.PauseGame();
     }
 
+    public void SaveGame()
```

Comment:
Misschien duidelijker als hier SaveMenu of SavePanel ofzo staat
Zelfde voor LoadGame

Reply:
Changed to SaveMenu and LoadMenu